tools/sgx-lkl-docker

```diff
@@ -44,22 +44,46 @@ function build() {
 FROM ubuntu:18.04
 RUN apt-get update && apt-get install -y ca-certificates
 
-FROM scratch
 # ca-certificates.crt is read by libcurl which is used by the Azure DCAP Client.
-COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-COPY . /
 
-# Note: WORKDIR also creates the directory.
-WORKDIR /tmp
-ENV TMPDIR=/tmp
+RUN groupadd -g 999 sgxlkl && \
+    useradd -g 999 -u 999 sgxlkl
```

Comment:
Is that the convention what uid/gid is used in containers?

Reply:
I've seen it elsewhere but there's not really a convention. It's likely that we change the uid later to match whatever the k8s plugin expects when bringing in the TAP device.